storm-core/src/clj/org/apache/storm/daemon/nimbus.clj

```diff
@@ -1793,16 +1793,16 @@
         (.mark nimbus:num-downloadChunk-calls)
         (check-authorization! nimbus nil nil "fileDownload")
         (let [downloaders (:downloaders nimbus)
-              ^BufferFileInputStream is (.get downloaders id)]
+              ^BufferInputStream is (.get downloaders id)]
```

Comment:
Is there a reason to change type hint class name?
http://storm.apache.org/releases/1.0.0/javadocs/org/apache/storm/utils/BufferFileInputStream.html
BufferFileInputStream is not a "is-a" BufferInputStream, and it also has close() method itself.